docs/ruleset_functions/are_all_terminal_heating_loops_attached_to_boiler.md

```diff
@@ -0,0 +1,26 @@
+# are_all_terminal_heating_loops_attached_to_boiler  
+
+**Description:** Returns TRUE if the fluid loop associated with the heating_from_loop associated with each terminal unit is attached to a boiler. Returns FALSE if this is not the case.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the heating_from_loop associated with each terminal unit is attached to a boiler.   
+- **terminal_unit_id_list**: List of terminal units to assess.  
+
+**Returns:**  
+- **are_all_terminal_heating_loops_attached_to_boiler**: Returns TRUE if the fluid loop associated with the heating_from_loop associated with each terminal unit is attached to a boiler. Returns FALSE if this is not the case.    
+ 
+**Function Call:** None  
+
+## Logic:   
+- Set are_all_terminal_heating_loops_attached_to_boiler equal to TRUE: `are_all_terminal_heating_loops_attached_to_boiler = TRUE`  
```

Comment:
does this need an applicability check to first confirm if terminal heating source = fluid loop?